CMakeLists.txt

```diff
@@ -59,6 +59,12 @@ set(
   CACHE PATH "Where to store downloaded files"
 )
 
+set(
+  FIREBASE_EXTERN_SOURCE_DIR
```

Comment:
Nit: elsewhere we spell out "external" (`download_external_sources`, `add_external_subdirectory`, `external/src`, etc). This stands out.

Reply:
Done.